### PR TITLE
[GH-803] Added release notes for 0.2.0

### LINKED ISSUE #803: add release notes for 0.2.0
include a description of what has changed from 0.1.0

you don't have to include installation and configuration, just reference https://github.com/aeternity/elixir-node/blob/master/docs/release-notes/release-notes-0.1.0.md

"Retrieve the software for running a node"
and "Running your node" should be included

@DanielaIvanova @Artur64 could you do this and then @cytadela8 add all the changes made in channels?

### PR DESCRIPTION
**Pull request by DanielaIvanova**
resolves #803

### CODE REVIEW

File: docs/release-notes/release-notes-0.2.0.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# About this release
+
+This is a maintenance release. It includes the following changes:
```
Comment (d-velev) [inline docs/release-notes/release-notes-0.2.0.md]: A lot of stuff was added, this doesn't seem like a maintenance release

File: docs/release-notes/release-notes-0.2.0.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+* Changed commitment hash calculations in naming system, to be `Hash(NameHash(name) + name_salt)` instead of `Hash(Hash(name + name_salt))`.
+* Added identifiers to `name` and `name_commitment` structures. 
+* Changed base58 encoding delimiter from `'$'` with `'_'`.
```
Comment (d-velev) [inline docs/release-notes/release-notes-0.2.0.md]: "to" instead of "with"

File: docs/release-notes/release-notes-0.2.0.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+* Upgraded IDNA library to introduce character and label validation according to IDNA 2008.
+* Removed `.aet` from allowed naming registrars.
+* Added `vm_version` field to `oracle` and `oracle_registration_tx` structures. 
+* Added `response_ttl` to `oracle_response_tx`.
+* Adjusted oracle serializations. Oracle query structure changes: added binary identifiers to senders and oracle owners.
+* Implemented functionality for garbage collecting of transactions in the pool. Transactions with outdated ttl are removed from the transaction pool.
```
Comment (d-velev) [inline docs/release-notes/release-notes-0.2.0.md]: ...Transactions with an outdated TTL...

File: docs/release-notes/release-notes-0.2.0.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+* Added functionality to get base gas price for each transaction.
+* Introduced Delegates in Channels
+* Added ChannelSnapshot transaction
+* Adjusted Channel structures for compatibility with v0.25.0 epoch: ChannelMutalCloseTx, ChannelStateOnChain, ChannelCreateTx
+* Adjusted verious Channels checks for compatibility with v0.25.0 epoch
```
Comment (d-velev) [inline docs/release-notes/release-notes-0.2.0.md]: various*

File: docs/release-notes/release-notes-0.2.0.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+Here, you can find [installation](https://github.com/aeternity/elixir-node/blob/master/docs/release-notes/release-notes-0.1.0.md#install-dependencies) and [configuration](https://github.com/aeternity/elixir-node/blob/master/docs/release-notes/release-notes-0.1.0.md#configuring-your-node) guides.
```
Comment (d-velev) [inline docs/release-notes/release-notes-0.2.0.md]: Here you can...

File: docs/release-notes/release-notes-0.2.0.md (hunk 1 above)
Comment (thepiwo) [reply docs/release-notes/release-notes-0.2.0.md]: What would you suggest? as there are various things changed I wouldn't give it a specific name. Maybe we should find a term for "catching up compatibility release"

Comment (d-velev) [reply docs/release-notes/release-notes-0.2.0.md]: Every release will be a compatibility catch up release, so we don't have different kinds of releases, maybe formulate  it like "As any of the previous, this release includes changes that improve compatibility with epoch nodes, more specifically:"

**Review by thepiwo — APPROVED** [review]

**Comment by thepiwo** [comment]
@d-velev can you approve now?

**Review by d-velev — APPROVED** [review]

**Pull request merged**